codegens/kotlin-okhttp/README.md

```diff
@@ -0,0 +1,74 @@
+# codegen-kotlin-okhttp
+
+> Converts Postman-SDK Request into code snippet for kotlin-okhttp.
+
+#### Prerequisites
+To run Code-Gen, ensure that you have NodeJS >= v6. A copy of the NodeJS installable can be downloaded from https://nodejs.org/en/download/package-manager.
```

Comment:
We recently bumped the requirements for this module to NodeJS 12. We can update the docs here to convey this information.